packages/services/src/workers/notification-sender.js

```diff
@@ -1,6 +1,7 @@
 import emailClient from '@aragonone/court-backend-shared/helpers/email-client'
 import { UserNotification } from '@aragonone/court-backend-server/build/models/objection'
 import * as notificationScanners from '../models/notification-scanners'
+import { accountData } from '../../../../emails/mock-utils'
```

Comment:
should we rename this to `utils` or `helpers`?

Reply:
renamed to helpers